### PR TITLE
Added more sample data

### PR DESCRIPTION
**Pull request by nianfei97**

### CODE REVIEW

**Comment by cjinting-nus** [comment]
Looks great, but the data is not changed for the new `Edit` command

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 public class Tag {
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
-    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+    public static final String VALIDATION_REGEX = ".+";
```
Comment (cjinting-nus) [inline src/main/java/seedu/address/model/tag/Tag.java]: Why the change to accept all characters though

Comment (nianfei97) [reply src/main/java/seedu/address/model/tag/Tag.java]: I guess just so that I can have dashes in my group names lol

Comment (cjinting-nus) [reply src/main/java/seedu/address/model/tag/Tag.java]: Because one of your invalid tags in your sample data was `friend*` but that's true I'll added dashes and underscores in but no special characters
